src/openzaak/components/catalogi/admin/admin_views.py

```diff
@@ -88,8 +95,17 @@ def get_context_data(self, **kwargs):
 
         iotypen = self.request.session.get("iotypen")
         if iotypen:
+            iot_dict = self.get_query_dict(
+                InformatieObjectType.objects.filter(catalogus=catalogus)
+                .order_by("omschrijving", "datum_begin_geldigheid")
+                .distinct("omschrijving")
```

Comment:
Hm I believe it will keep the first row, i.e. the earliest version

Reply:
That indeed was the case and the tests were wrong.